src/Agent.Listener/Configuration/LinuxServiceControlManager.cs

```diff
@@ -100,37 +100,31 @@ public override void StartService(string serviceName)
                 ChangeOwnershipToLoginUser(filesToChange);
                 ReloadSystemd();
                 ExecuteSystemdCommand("start " + serviceName);
+                _term.WriteLine(StringUtil.Loc("ServiceStartedSuccessfully", serviceName));
             }
             catch (Exception)
             {
-                _term.WriteError(StringUtil.Loc("LinuxServiceStartFailed"));
+                _term.WriteError(StringUtil.Loc("CanNotStartService"));
                 throw;
             }
         }
 
-        protected virtual string GetUnitFile(string serviceName)
+        public override void StopService(string serviceName)
         {
-            return Path.Combine(SystemdPathPrefix, serviceName);
-        }
-
-        protected virtual bool CheckIfSystemdExists()
-        {
-            Trace.Info(nameof(CheckIfSystemdExists));
+            Trace.Info(nameof(StopService));
```

Comment:
I dont see a need to add a optional bool param to entering. Rather the entering should use Info bydefault? Whats the default log level? IMO we should keep it to verbose if we want to capture all logs. Then I can just call to entering and need not change the default behaviour of entering.